app/design/frontend/base/default/template/somethingdigital/criteo/basket.phtml

```diff
@@ -2,15 +2,32 @@
 
 <?php
     echo $this->getChildHtml('somethingdigital_criteo_device_detection');
+    $email = Mage::helper('somethingdigital_criteo')->getHashedEmail();
+    $items = $this->getCriteoViewBasket();
+    $useDataLayer = Mage::getStoreConfig('somethingdigital_criteo/settings/use_datalayer');
 ?>
-
-<script type="text/javascript">
-    window.criteo_q = window.criteo_q || [];
-    window.criteo_q.push(
-        { event: "setAccount", account: "<?php echo Mage::getStoreConfig('somethingdigital_criteo/settings/account_number'); ?>" },
-        { event: "setHashedEmail", email: ["<?php echo Mage::helper('somethingdigital_criteo')->getHashedEmail();?>"] },
-        { event: "setSiteType", type: deviceType },
-        { event: "viewBasket", item: <?php echo $this->getCriteoViewBasket(); ?> }
-    );
-</script>
-<?php endif; ?>
\ No newline at end of file
+    <?php if (!$useDataLayer): ?>
```

Comment:
Would rather do `if ($useDataLayer)` than `if (!$useDataLayer)`

Reply:
How strongly do you think this change is required? It would mean i would have to rewrite the templates once again.

Reply:
You on't have to change it.